bluetooth_mesh/models/models.py

```diff
@@ -1498,6 +1495,49 @@ async def request():
             request, retransmissions=retransmissions, send_interval=send_interval
         )
 
+    async def get_lightness_range(
```

Comment:
How does `LightRangeCommand` know about this coro?

Reply:
`ModelGetCommandMixin`

Reply:
https://github.com/SilvairGit/python-bluetooth-mesh/blob/2bd0fbd6188e3fdd7bed4457795eb0f84e8abf40/bluetooth_mesh/apps/meshcli.py#L317-L335

Reply:
Thanks.